### PR TITLE
Creating a pull request

### PR DESCRIPTION
**Pull request by rohittm**
# Changes:

1. Changing indexer to fetch metadata props like keywords, subject, and usage terms, removing color tagging.

2. Built UI using React Js

### CODE REVIEW

**Comment by abdul** [comment]
Please update README to have at least instructions to run this code. Please update PR to have list of changes done so far.

File: src/components/searchbox.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import elasticsearch from 'elasticsearch'
+
+let client = new elasticsearch.Client({
+	host: 'localhost:9200',
```
Comment (abdul) [inline src/components/searchbox.js]: better way than hard coding host:port?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "homepage": "http://rohittm.github.io/Searcher"
```
Comment (abdul) [inline package.json]: I am not sure what's right way, please figure out what do people put in homepage in this kind of set up (fork --> PR)?

File: src/components/single.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	} 
+
+    render () {
+		return (
+			<div className="search_results">
+				<hr />
+                <ul>
+				{ this.props.results.map((result) => {
+					return (
+						<img className="image" src={"http://rohitmotwani.com/photos/" + result._source.file_name} alt={result._source.file_name + i++} />
```
Comment (abdul) [inline src/components/single.js:18]: Why do we need this? Aren't we using a docker based stack here?

File: src/components/searchbox.js (hunk 1 above)
Comment (rohittm) [reply src/components/searchbox.js]: The Documentation at [Elastic Search Js API](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/quick-start.html) suggested this way of using the API. and I guess whenever we use elasticsearch server, it uses the same host and port until we change it in the hosts file isn't it ?

File: package.json (hunk 2 above)
Comment (rohittm) [reply package.json]: I have removed the homepage for now, It was when I published the application on github pages

File: src/components/single.js (hunk 3 above)
Comment (rohittm) [reply src/components/single.js:18]: Are you talking about my FTP server ? If yes, I needed a location to store the jpeg images as locally I was'nt able to locate images through the src attribute. I used {app-host:port + result._source.file_name} initially.

Comment (rohittm) [reply src/components/single.js:18]: I believe i hardcoded it, I needed it for testing and I'm not unable to link it in Local node server.

File: src/components/searchbox.js (hunk 1 above)
Comment (abdul) [reply src/components/searchbox.js]: In real scenario, client app would look at some host instead of localhost (which is fine for development).

Comment (rohittm) [reply src/components/searchbox.js]: We can change host accordingly when real time ? or should I add an alias and then change it in Client's Hosts file ?

**Pull request merged**